iroh-net/src/key/node.rs

```diff
@@ -106,7 +106,7 @@ pub struct SecretKey(crypto_box::SecretKey);
 
 impl From<crate::tls::SecretKey> for SecretKey {
     fn from(key: crate::tls::SecretKey) -> Self {
-        SecretKey(crypto_box::SecretKey::from(key.to_scalar_bytes()))
+        SecretKey(crypto_box::SecretKey::from(key.to_scalar()))
```

Comment:
this is depending on https://docs.rs/crypto_box/0.9.0/crypto_box/struct.SecretKey.html#impl-From%3CScalar%3E-for-SecretKey
so I'm not sure what's wrong